Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2021"
 [workspace]
 members = ["state-reconstruct-fetcher", "state-reconstruct-storage"]
 
+[profile.release]
+panic = 'abort'
```

Comment:
I totally agree the threads shouldn't panic - if there is code that panics on invalid network data, disk failure or whatever we should fix that. But I don't know where the code might be; https://github.com/eqlabs/zksync-state-reconstruct/issues/104 happened just once - it could be some irreproducible cosmic ray... This change is just to find it.